zebra-consensus/src/transaction/tests.rs

```diff
@@ -660,3 +660,24 @@ fn mock_sprout_join_split_data() -> (JoinSplitData<Groth16Proof>, ed25519::Signi
 
     (joinsplit_data, signing_key)
 }
+
+#[test]
+fn empty_sprout_pool_after_nu() {
```

Comment:
Confusing name: Please use a name that describes which parts of sprout are disabled.